COA-733: Add Free Apps Archival documentation (#4071)
mendix/docs: content/developerportal/deploy/mendix-cloud-deploy.md

You can upgrade a Free App to a licensed node with a *node* in the Mendix Cloud. Instructions for doing this are here: [Licensing Mendix Cloud Apps](licensing-apps).

#### 1.1.1 Free Apps Archival{$free-apps-archival}

> If a Free App has been in Sleep Mode for three months or longer, the Technical Contact will be informed that the Free App will be archived two weeks later. The project and model will be retained, but all data in the Free App node will be removed.

Reviewer: We don't really talk about "Nodes" for Free Apps. How about:
```suggestion
If a Free App has been in Sleep Mode for three months or longer, the Technical Contact will be informed that the Free App will be archived in another two weeks. The project and model will be retained, but all data for the Free App will be removed.
```

Reviewer 2: Yes, this is definitely better. Although, do you think the last clause, "but all data for the Free App will be removed." is clear enough, specifically does it make the distinction between us deleting the database/files/app node, but we keep the model/project?

Reviewer 2: I guess we could say:

"The project and model will be retained, but the application, database, and files for the Free App will be removed."

Reviewer: How about "deleted" rather than "removed"?

There are two ways to avoid Free Apps Archival:

1. Visit the Free App.

> 2. Deploy a new model to your Free App node.

Reviewer: Does this have to be a **new** app?
```suggestion
2. Deploy an app to your Free App environment.
```

Reviewer 2: We wrote "deploy a new model" because there is a 1:1 mapping from (free) app to (free) environment, unlike in the paid cloud where you can deploy different apps to any node you like.

(At least, that's the standard way of doing things, via Studio).

We could go with "a new version of your app", does that work better?

Reviewer: Yep - a new version of your app sounds good.
